src/hooks/useLogin.js

```diff
@@ -18,37 +19,66 @@ export const useLogin = () => {
   const clearAll = useCallback(() => {
     setState(initialState);
     setError("");
+    setLoading(false);
   }, []);
 
   const onLogin = useCallback(
     async (event) => {
       event.preventDefault();
+      setError("");
```

Comment:
What error will be shown if you click login button without entering both email and password?